Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 @Library('sec_ci_libs@v2-latest') _
 
-def master_branches = ["master", ] as String[]
+def master_branches = ["releases/0.4", "master", ] as String[]
```

Comment:
we need `releases/0.3` as well

Reply:
Honestly, I didn't need to add those other branches on Marathon because the Jenkinsfiles don't have the security plugin specified, and Community PRs use the Jenkinsfile on master.

Reply:
But not a big deal, I'll add it.